auth/src/main/java/org/aerogear/mobile/auth/configuration/KeycloakConfiguration.java

```diff
@@ -0,0 +1,80 @@
+package org.aerogear.mobile.auth.configuration;
+
+import android.net.Uri;
+import android.support.annotation.VisibleForTesting;
+
+import org.aerogear.mobile.core.configuration.ServiceConfiguration;
+
+/**
+ * A class to represent the configuration options of the Keycloak service
+ */
+public class KeycloakConfiguration {
+
+    private static final String SERVER_URL_NAME = "auth-server-url";
+    private static final String REALM_ID_NAME = "realm";
+    private static final String CLIENT_ID_NAME = "clientId";
```

Comment:
I think this should be `CLIENT_ID_NAME = "resource"`.

Reply:
this will be fixed in another PR